### ISSUE TITLE
cluster size

### ISSUE THREAD

**Issue by anurag10** [inikdom/rnn-speech#44]
can I know the cluster size you have used?

**Comment by AMairesse** [comment]
Hi, I'm not sure to understand what you mean by "cluster size". Are you talking about the number of samples used together for one step of training ? If this is the case the english model was trained with a batch size of 10 files and a mini-batch of 3. That mean that the train step (updating the network) was based on the error found on 30 files each time. I found that it was a good compromise between the stability of the training and the performance.
Is that what you were asking ?

**Comment by anurag10** [comment]
Thanks for your answer. Actually I wanted to know if you run it on single machine or on multiple machines?

**Comment by AMairesse** [comment]
Yes it's running on a single machine : training and/or inference (a good GPU is highly recommended)
You could probably use a machine with multiple GPU because tensorflow would manage it but I'm not even sure it would be efficient. I mean the code would probably not make a optimal use of it.
And about multiple machine I haven't look in supporting it. It could be done but would require quite a lot of changes.
